### PR TITLE
fix constraints

### PR DESCRIPTION
**Pull request by synchuk2010**
### Description
- Fix constraints on welcome screen

### CODE REVIEW

**Comment by ghost** [comment]
## Black Duck Security Report

Merging #104 into develop will not change security risk.

[Click here to see full report](https://copilot.blackducksoftware.com/github/repos/essentiaone/Essentia-iOS/pull-requests/104)

File: Modules/EssUI/Sources/TableAdapterCore/TableAdapterHelper.swift [hunk 1]
```diff
@@ -78,7 +78,7 @@ public class TableAdapterHelper {
         case .centeredButton:
             return 75.0
         case .attributedCenteredButton:
-            return 40.0
+            return 45.0
```
Comment (impul) [inline Modules/EssUI/Sources/TableAdapterCore/TableAdapterHelper.swift]: Calculate button height due to component title text

File: Modules/EssUI/Sources/TableAdapterCore/TableComponentCells/CenteredButton/TableComponentCenteredButton.swift [hunk 2]
```diff
@@ -15,6 +15,8 @@ class TableComponentCenteredButton: UITableViewCell, NibLoadable {
     
     @IBOutlet weak var rightInset: NSLayoutConstraint!
     @IBOutlet weak var leftInset: NSLayoutConstraint!
+    @IBOutlet weak var topInset: NSLayoutConstraint!
```
Comment (impul) [inline Modules/EssUI/Sources/TableAdapterCore/TableComponentCells/CenteredButton/TableComponentCenteredButton.swift]: It is not necessary

**Review by impul — CHANGES_REQUESTED** [review]

File: Modules/EssUI/Sources/TableAdapterCore/TableComponentCells/CenteredButton/TableComponentCenteredButton.swift (hunk 2 above)
Comment (synchuk2010) [reply Modules/EssUI/Sources/TableAdapterCore/TableComponentCells/CenteredButton/TableComponentCenteredButton.swift]: removed

File: Modules/EssUI/Sources/TableAdapterCore/TableAdapterHelper.swift (hunk 1 above)
Comment (synchuk2010) [reply Modules/EssUI/Sources/TableAdapterCore/TableAdapterHelper.swift]: see, b802f32

File: Modules/EssUI/Sources/TableAdapterCore/TableAdapterHelper.swift [hunk 3]
```diff
@@ -77,8 +77,8 @@ public class TableAdapterHelper {
         case .borderedButton: fallthrough
         case .centeredButton:
             return 75.0
-        case .attributedCenteredButton:
-            return 40.0
+        case .attributedCenteredButton(let attributedTitle, _, _, _):
+            return (attributedTitle?.height(with: tableView.bounds.width))! + 30
```
Comment (impul) [inline Modules/EssUI/Sources/TableAdapterCore/TableAdapterHelper.swift]: Please do not do force unwrapping

**Review by impul — CHANGES_REQUESTED** [review]

**Review by impul — APPROVED** [review]

**Pull request merged**